### ISSUE TITLE
Mumble 1.3 Occasionally Becomes Painfully Staticky

### ISSUE THREAD

**Issue by freddwo** [mumble-voip/mumble#4297]
Mumble occasionally becomes incredibly staticky. Other people's voices become painfully staticky and loud and apparently my own voice sounds that way to others as well. As far as I can tell, I am not doing anything that should cause this, just pressing my push to talk, and I also don't know how to fix it. Eventually if I wait long enough it just goes back to functioning normally, without changing anything.

**Steps to Reproduce**
No clue. It seems to just *happen*. I'm sorry this is so unhelpful but my confusion about the cause is a major reason I'm even making this post.

 - OS: Latest version of Solus Linux distro
 - Version: Mumble 1.3

Sorry this post is rather vague. I'm really quite confused and don't know much about what might be going on. I'm happy to answer any questions I can. Any advice is appreciated. Thanks in advance.

**Comment by Krzmbrzl** [comment]
Have you tried using other recording software to record audio with for the same amount of time it usually takes for your audio in Mumble to become distorted? Just to make sure it's not actually the hardware that is causing trouble.

What audio interface are you using? PulseAudio? Did you already try out a different one?

Does the problem occur occasionally or rather after amount of time X of being on a server, etc... ? 

Do these problems occur on a specific server only or does it happen on multiple different servers?

**Comment by TerryGeng** [comment]
Are you sure this problem only happens to mumble?

Based on your description, I think maybe it is caused by some misconfiguration of your system's audio settings.
As far as I've known, the code used to process audio input and output is separated. I can't immediately come up with a reason that this problem happens to the input and output at the same time.

**Comment by freddwo** [comment]
> Have you tried using other recording software to record audio with for the same amount of time it usually takes for your audio in Mumble to become distorted?

I have spent several hours in Discord without an issue, but I haven't tried switching to a different program while the static was happening. I'll try that next time. 

> What audio interface are you using? PulseAudio? Did you already try out a different one?

I have been using PulseAudio. I tried switching to OSS but then couldn't hear anyone or talk to anyone. ALSA seems to work, but I haven't used it for an extended period of time yet.

> Does the problem occur occasionally or rather after amount of time X of being on a server, etc... ?

I haven't timed the intervals that it happens, but it seems to be more occasionally than after a certain amount of time just based off of my estimates. Also the problem resolves itself at different times. Sometimes it lasts hours. Sometimes only 15-20 min.

> Do these problems occur on a specific server only or does it happen on multiple different servers?

Should've thought of this before, but it has only happened in one server so far. The only server I spend a lot of time in. Next time it gets staticky I'll try moving to a different server to see if the problem persists.

Thanks for you questions and suggestions @Krzmbrzl

**Comment by freddwo** [comment]
> Are you sure this problem only happens to mumble?

I am not *certain* but while Mumble was staticky, I think I have been able to use in-game voice for videogames properly. Well, at least no one complained about any static while using in-game. I should test this more definitively. I have spent several hours in Discord without an issue.

> Based on your description, I think maybe it is caused by some misconfiguration of your system's audio settings.

This seems entirely possible, especially considering this never happened before my recent switch to Linux. I am using a Blue Snowball mic plugged into a USB slot on my CPU. I'll look into proper mic settings for this mic.

Right now I'm using ALSA instead of PulseAudio to see if that fixes anything, then I'll try other things if this doesn't work.

Thanks @TerryGeng . I'll post an update as soon as I have an update to post.
